### PR TITLE
[HPRO-595] Handle centrifuge type for kit orders

### PR DESCRIPTION
**Pull request by shyamnaru**

### CODE REVIEW

File: src/Pmi/Controller/BiobankController.php [hunk 1]
```diff
@@ -194,7 +194,11 @@ public function orderAction($biobankId, $orderId, Application $app, Request $req
         $steps = ['collect', 'process', 'finalize'];
         // Set default step if current step not exists in the available steps
         $currentStep = !in_array($order->getCurrentStep(), $steps) ? 'collect' : $order->getCurrentStep();
-        $finalizeForm = $order->createBiobankOrderFinalizeForm($app['form.factory']);
+        $site = $app['em']->getRepository('sites')->fetchOneBy([
+            'deleted' => 0,
+            'google_group' => $app->getSiteId()
```
Comment (jt2k) [inline src/Pmi/Controller/BiobankController.php:199]: Don't we want the order's created site here?

File: src/Pmi/Controller/BiobankController.php [hunk 2]
```diff
@@ -225,13 +229,33 @@ public function orderAction($biobankId, $orderId, Application $app, Request $req
...
+                        }
+                        if ($order->get('type') === 'kit' && empty($order->get('processed_centrifuge_type'))) {
+                            $site = $app['em']->getRepository('sites')->fetchOneBy([
+                                'deleted' => 0,
+                                'google_group' => $app->getSiteId()
+                            ]);
+                            if (!empty($site['centrifuge_type'])) {
+                                $order->set('processed_centrifuge_type', $site['centrifuge_type']);
+                            }
+                        }
```
Comment (jt2k) [inline src/Pmi/Controller/BiobankController.php:248]: 1. This if clause is the same as above. Not sure why it needs to be in a separate block.
2. Don't you already have the site retrieved from the database from above?

Comment (shyamnaru) [reply src/Pmi/Controller/BiobankController.php]: Sorry, forgot to remove some of the unwanted code that I have written previously.

**Comment by shyamnaru** [comment]
Also notice that in we should be getting the mayo account number using the current site id when `sendOrderToMayo` method is called in Order Controller (which I changed yesterday to fix the issue for biobank users).

**Pull request merged**